### DISCUSSION TITLE
Initializing dev->pix_format.width and dev->pix_format.height to zero

### DISCUSSION THREAD

**Discussion by Alienmaster** [umlaeute/v4l2loopback#538] [General]
Hi,

while reviewing the v4l2loopback code we noticed the following initialization:

https://github.com/umlaeute/v4l2loopback/blob/d01c20cca58772e7b29faef6da393c919a4d531b/v4l2loopback.c#L2523-L2524

As both lines are initialized to `0` instead of the commented-out value we would have expected, we'd like to know if there's a special reason for why both values are initialized to `0` instead of `V4L2LOOPBACK_SIZE_DEFAULT_WIDTH` or `_max_width` and `V4L2LOOPBACK_SIZE_DEFAULT_HEIGHT` or `_max_height` respectively?

**Comment by umlaeute** [comment]
honestly i cannot remember.

however: what is wrong with `0`?
it seems to be better to report something invalid than to pretend that there's a set size (while it really isn't)

**Comment by umlaeute** [comment]
as of 93c17d2e2f95780228b297500bd9d8e729ff7288, the loopback device is again populated with some default framesize
